Sources/UIComponents/Feedback.swift

```diff
@@ -0,0 +1,255 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nicolai Dam on 13/02/2023.
+//
+
+import SwiftUI
+
+public struct FeedbackStyling {
+    
+    let primaryButtonColor: Color
+    let secondaryColor: Color
+    let bodyTextColor: Color
+    let borderColor: Color
+    let backgroundColor: Color
+    
+    public init(
+        primaryButtonColor: Color = Color(#colorLiteral(red: 0, green: 0.5659442544, blue: 0.287532568, alpha: 1)),
+        secondaryColor: Color = Color(#colorLiteral(red: 0.1018853113, green: 0.1138664857, blue: 0.3018276095, alpha: 1)),
+        bodyTextColor: Color = Color(#colorLiteral(red: 0.262745098, green: 0.262745098, blue: 0.3882352941, alpha: 1)),
+        borderColor: Color = Color(#colorLiteral(red: 0.262745098, green: 0.262745098, blue: 0.3882352941, alpha: 1)),
+        backgroundColor: Color = Color(#colorLiteral(red: 1, green: 0.9999999404, blue: 0.9999999404, alpha: 1))
```

Comment:
Can you make an Enum for these colors like font and reuse those instead?

Reply:
Sure i can do that

Reply:
I dont know if it makes sense since the colors are specifically from the Ejerskifte color palette? Then I would have to name it as that?

Reply:
Made a Color extension with the custom colors and one for images as well